pyproject.toml

```diff
@@ -0,0 +1,56 @@
+[build-system]
+# Defined by PEP 518
+requires = [
+  "setuptools>=42",
+  "setuptools_scm[toml]>=6.0",
+  "wheel",
+]
+# Defined by PEP 517
+build-backend = "setuptools.build_meta"
+
+[tool.black]
+line-length = 79
+target-version = ['py37', 'py38', 'py39']
+include = '\.pyi?$'
+exclude = '''
+
+(
+  /(
+      \.eggs         # exclude a few common directories in the
+    | \.git          # root of the project
+    | \.hg
+    | \.mypy_cache
+    | \.tox
+    | \.venv
+    | _build
+    | buck-out
+    | build
+    | compiled_krb
+    | dist
+    | etc
+    | iris_image_test_output
```

Comment:
Should this be here?

Reply:
Nice one... totally guilty of cut'n pasting :rofl:

Reply:
Done